Shoko.Plugin.Abstractions/Attributes/RenamerAttribute.cs

```diff
@@ -19,4 +20,4 @@ public string Description
             set => _desc = value;
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
I don't like the removal of the trailing line endings, but I also won't force you to redo the PR to remove the changes. Same with the BOM(?) changes at the start of the files.

Reply:
I will look at it Visual Studio 2022 just does it by itself but it also says `User preferences overridden by .editorconfig` so I assumed those formatting changes came from there.

Reply:
Adding extra line at the end is set in `.editorconfig` https://github.com/ShokoAnime/ShokoServer/blob/2451fb2591f696c2793327a162e9af08b41c756c/.editorconfig#L93 and BOM shouldn't be there according to that as well, maybe that one file had a BOM?